src/burgers1d/main.cpp

```diff
@@ -0,0 +1,244 @@
+// Copyright (C) 2017-2018  Waterloo Quantitative Consulting Group, Inc.
+// See COPYING and LICENSE files at project root for more details.
+
+/** \mainpage blitzdg documentation
```

Comment:
Don't copy and paste the main page blob to multiple spots. Should probably start making separate pages for separate solvers.